server/lib/systemd/pbench-server.service.rhel7

```diff
@@ -9,8 +9,10 @@ Documentation = https://github.com/distributed-system-analysis/pbench
 After=network.target httpd.service
 
 [Service]
+user = $(/opt/pbench-server/bin/pbench-config -l user pbench-server)
```

Comment:
How does `pbench-config` get the `_PBENCH_SERVER_CONFIG` environment variable defined?

Reply:
`pbench-config` script internally tries to read the environment variable  `_PBENCH_SERVER_CONFIG` It needs to be exported first I think

Reply:
After little digging it seems like the environment variable we set in the systemd service file with either `ENVIRONMENT` or `EnvironmentFIle` parameter can only be used with `Exec*` parameter  (like in `ExecStart` or `ExecStop`). I think I can just use the user and group as `pbench` since the service runs as `pbench` user. Let me know if thats fine

Reply:
So we might want to merge the server ansible playbooks and roles first.  That way, the service file can be a jinja2 template which we render with the variables the playbooks and roles determine.